test/parallel/test-net-server-listen-options.js

```diff
@@ -66,6 +70,13 @@ const listenOnPort = [
                       name: 'TypeError',
                       message: /^The argument 'options' must have the property "port" or "path"\. Received .+$/,
                     });
+    } else if (typeof options.host === 'string' && !options.host.match(/^[a-zA-Z0-9-:%.]+$/)) {
```

Comment:
Can we use the regex mentioned 147. line instead of writing it again?

Reply:
These are two different files